### DISCUSSION TITLE
Trying to read variant.sku data on dynamic route with swell.product.get({product_slug}) method...?

### DISCUSSION THREAD

**Discussion by slanelb** [swellstores/community#151] [Help] [answered]
Hi, I wonder if anyone would be able to help me?
I'm getting 'undefined' back on my frontend when reading `variant.sku`. This is within a dynamic route using the swell js `swell.product.get({product_slug})` method.
Doesn't seem to be an issue with other adjacent data and no problem getting the `variant.sku` data using the `swell.products.list()` method on another page.
Any help would be greatly appreciated.

**Comment by sags95** [comment, marked as answer]
@slanelb You should be able to get this field when passing in `expand: ['variants']` like:

```
await swell.products.get({product_slug}, {
      expand: ['variants']
    });
```

**Reply by slanelb** [reply]
Thank you for replying. I've since moved over to the graphql. Seems to be going ok at the moment.
